resources/views/partials/_header.blade.php

```diff
@@ -1,7 +1,9 @@
-<nav>
-    <ul>
-        <li><a href="/">Home</a></li>
-        <li><a href="/users">Users</a></li>
-        <li><a href="/tasks">Tasks</a></li>
-    </ul>
+<nav class="ml-0 m-4">
+    <div class="flex">
+        <a href="/"><h1 class="text-3xl font-bold">2DO</h1></a>
+        <ul class="list-none flex items-center ml-auto mr-auto">
+            <li class="ml-3"><a href="/users">Users</a></li>
+            <li class="ml-3"><a href="/tasks">Tasks</a></li>
+        </ul>
+    </div>
 </nav>
```

Comment:
![image](https://user-images.githubusercontent.com/47534548/235314675-a91eb045-8243-455b-ad56-fe7e3efd3f0a.png)

add paddin for `x`